Dockerfile

```diff
@@ -1,13 +1,64 @@
-FROM python:latest
+# Global ARG for all stage
+ARG REQUIREMENTS_PATH=./app/requirements.txt
```

Comment:
There is no ` ./app/requirements.txt` file in your PR. If you want to package the dependencies there, please create the necessary file and directory structure.